scripts/integrity-check.sh

```
@@ -0,0 +1,413 @@
+# Load env
+if [ -f "$WORKSPACE/.env" ]; then
+  set -a
+  eval "$(grep -E '^(INTEGRITY_|GEMINI_API_KEY|GOOGLE_API_KEY|LLM_API_KEY)' "$WORKSPACE/.env" 2>/dev/null)" || true
+  set +a
+fi
+
+# Config defaults (can be overridden by integrity.yaml or env)
+INTEGRITY_ENABLED="${INTEGRITY_ENABLED:-true}"
+SAMPLE_N="${INTEGRITY_SAMPLE_N:-10}"
+GLOBAL_THRESHOLD="${INTEGRITY_THRESHOLD:-0.85}"
+REFLECTOR_THRESHOLD="${INTEGRITY_REFLECTOR_THRESHOLD:-$GLOBAL_THRESHOLD}"
```

Shellcheck (0.11.0)

[warning] REFLECTOR_THRESHOLD appears unused. Verify use (or export if used externally).

(SC2034)